python/lsst/cp/verify/verifyDark.py

```diff
@@ -68,12 +69,13 @@ def verify(self, exposure, statisticsDict):
         success : `bool`
             A boolean indicating if all tests have passed.
         """
-        detector = exposure.getDetector()
         ampStats = statisticsDict['AMP']
         metadataStats = statisticsDict['METADATA']
 
         verifyStats = {}
         success = True
+        gains = getExposureGains(exposure)
+        readNoises = getExposureReadNoises(exposure)
```

Comment:
Same comments here across verifyDark as above.

Reply:
Edited in parallel so they should have identical changes, but I'll carefully check when I commit.